ckanext/dcatapit/harvesters/csw_harvester.py

```diff
@@ -281,3 +370,43 @@ def get_package_dict(self, iso_values, harvest_object):
 
         # End of processing, return the modified package
         return package_dict
+
+
+    def get_agent(self, agent_type, agent_string, default_values):
```

Comment:
We have this method in utils.py, however this is not needed here as specified in the above comments. Please remove this piece of code.